runtime/include/chplmath.h

```diff
@@ -33,32 +33,32 @@ static inline float chpl_macro_NAN(void) {
-static inline int chpl_macro_float_isnan(float x) { return isnan(x); }
-static inline int chpl_macro_double_signbit(double x) { return signbit(x); }
-static inline int chpl_macro_float_signbit(float x) { return signbit(x); }
+MAYBE_GPU static inline int chpl_macro_double_isinf(double x) { return isinf(x); }
+MAYBE_GPU static inline int chpl_macro_float_isinf(float x) { return isinf(x); }
+MAYBE_GPU static inline int chpl_macro_double_isfinite(double x) { return isfinite(x); }
+MAYBE_GPU static inline int chpl_macro_float_isfinite(float x) { return isfinite(x); }
+MAYBE_GPU static inline int chpl_macro_double_isnan(double x) { return isnan(x); }
+MAYBE_GPU static inline int chpl_macro_float_isnan(float x) { return isnan(x); }
+MAYBE_GPU static inline int chpl_macro_double_signbit(double x) { return signbit(x); }
+MAYBE_GPU static inline int chpl_macro_float_signbit(float x) { return signbit(x); }
 
 // 32-bit Bessel functions aren't available on all platforms. For cases where
 // we know they're available use them since they should be faster, but in other
 // cases default to using the 64-bit versions and casting.
 #ifdef __linux__
-static inline float chpl_float_j0(float x) { return j0f(x); }
-static inline float chpl_float_j1(float x) { return j1f(x); }
-static inline float chpl_float_jn(int n, float x) { return jnf(n, x); }
-static inline float chpl_float_y0(float x) { return y0f(x); }
-static inline float chpl_float_y1(float x) { return y1f(x); }
-static inline float chpl_float_yn(int n, float x) { return ynf(n, x); }
+MAYBE_GPU static inline float chpl_float_j0(float x) { return j0f(x); }
+MAYBE_GPU static inline float chpl_float_j1(float x) { return j1f(x); }
+MAYBE_GPU static inline float chpl_float_jn(int n, float x) { return jnf(n, x); }
+MAYBE_GPU static inline float chpl_float_y0(float x) { return y0f(x); }
+MAYBE_GPU static inline float chpl_float_y1(float x) { return y1f(x); }
+MAYBE_GPU static inline float chpl_float_yn(int n, float x) { return ynf(n, x); }
 #else
-static inline float chpl_float_j0(float x) { return (float)j0(x); }
-static inline float chpl_float_j1(float x) { return (float)j1(x); }
-static inline float chpl_float_jn(int n, float x) { return (float)jn(n, x); }
-static inline float chpl_float_y0(float x) { return (float)y0(x); }
-static inline float chpl_float_y1(float x) { return (float)y1(x); }
-static inline float chpl_float_yn(int n, float x) { return (float)yn(n, x); }
+MAYBE_GPU static inline float chpl_float_j0(float x) { return (float)j0(x); }
+MAYBE_GPU static inline float chpl_float_j1(float x) { return (float)j1(x); }
+MAYBE_GPU static inline float chpl_float_jn(int n, float x) { return (float)jn(n, x); }
+MAYBE_GPU static inline float chpl_float_y0(float x) { return (float)y0(x); }
+MAYBE_GPU static inline float chpl_float_y1(float x) { return (float)y1(x); }
+MAYBE_GPU static inline float chpl_float_yn(int n, float x) { return (float)yn(n, x); }
 #endif
 
 #ifdef DEFINE_32_BIT_MATH_FNS
```

Comment:
Also in the spirit of keeping the functions in this file consistent, I think these versions can be removed. It looks like they were sunos specific